misc/config.yaml

```diff
@@ -4,6 +4,9 @@ guild-ids:
   - 129409120491092041
   - 1290192091029091120
 
+cache:
+  size: 16
```

Comment:
Any reason the cache is so small? I don't see this getting many hits

Reply:
Yeah I should split it into two sizes. One for events, where if there's going to be a hit, that'll likely be the only one, and a much larger one for foreign key maintenance, like making sure members are up-to-date.